server/fishtest/api.py

```diff
@@ -347,6 +347,82 @@ def get_elo(self):
         run["elo"] = a
         return run
 
+    @view_config(route_name="api_calc_elo")
```

Comment:
DEV running with:
```diff
diff --git a/server/fishtest/api.py b/server/fishtest/api.py
index 6af5a76..a7b2ed4 100644
--- a/server/fishtest/api.py
+++ b/server/fishtest/api.py
@@ -348,9 +348,9 @@ class ApiView(object):
         return run

     @view_config(route_name="api_calc_elo")
-    def calc_elo(self, api):
+    def calc_elo(self):
         self.__t0 = datetime.utcnow()
-        self.__api = api
+        self.__api = "/api/calc_elo"

         W = self.request.params.get("W")
         D = self.request.params.get("D")
```

Reply:
Works fine but is hardcoding the API this way necessary or is it the only way that works like this?

Reply:
The workers API's use this pattern, not so diffrent from hardcoding
https://github.com/glinscott/fishtest/blob/3cb76834a6cabc72ab479f13f5b74190fc1bf35e/server/fishtest/api.py#L435C1-L437C51

Reply:
> The workers API's use this pattern, not so different from hardcoding
> 
> https://github.com/glinscott/fishtest/blob/3cb76834a6cabc72ab479f13f5b74190fc1bf35e/server/fishtest/api.py#L435-L437

Works fine with me. Now probably we can enable the nginx cache, what do you think?

Reply:
DEV updated.
The nginx cache is a task for real man ;)